UI_Engine/Query/SearchItems.cs

```diff
@@ -107,7 +107,6 @@ public static IEnumerable<Type> TypeItems()
         {
             return Engine.Reflection.Query.BHoMTypeList()
                 .Concat(Engine.Reflection.Query.BHoMInterfaceList())
-                .Concat(new List<Type> { typeof(object), typeof(Type), typeof(Enum), typeof(bool), typeof(byte), typeof(char), typeof(double), typeof(int), typeof(string) })
```

Comment:
Not to sure why you are getting rid of these? Think we want them for generating dictionaries of specific types etc? I know I have used them at least a few times.

Reply:
The main reason why I removed these is because the local menu is completely wrong if we include them. But I agree, that was bad choice, I'll try to fix the menu instead.

Reply:
How do you feel with this? It looks alright to me
![image](https://user-images.githubusercontent.com/26899347/60346133-6c32ae80-99b2-11e9-9e0b-712032e64b67.png)

Reply:
I would agree with that solution

Reply:
So, this solution creates other problems, and since it is not due to solve it in this pr, I will leave it as it is in the master right now